UnitTesting-master/src/test/java/testNG/CartTestNG.java

```diff
@@ -0,0 +1,62 @@
+package testNG;
+
+import com.github.javafaker.Faker;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.*;
+
+public class CartTestNG {
+
+    private static final Faker faker = new Faker();
+    public static String cartForTestName = faker.business().creditCardNumber();
```

Comment:
Make it private.
I recommend to make it non-static, because if you will run tests in parallel, then one name will be used by all carts